### ISSUE TITLE
On top of the world (No hacks)

### ISSUE THREAD

**Issue by tutuototo** [Wynncraft/Issues#8139]
I really do not know if this has happened to anyone else, however this looks like a pretty rare bug...

Minecraft IGN: tutuototo

Bug: I've disconnected from WynnCraft since my modpack seemed to bug out a little (WynnCraft Enhanced). When I rejoined, I spawned on top of the barrier blocks on the top of the world. I don't think I have a way to get back down. I have filmed evidence on my phone if it's needed.

To go in a little more details, I was heading for Lusuco from Nesaak. I was fighting Corrupt Sentinels (not sure about the name) in the tunnel. I saw the names of the items weren't popping up (Rare Bow 48-52, for example), so I decided to put myself in a 2-block high area at the entrance of the cave, then I got off. When I got back, I was on top of the barrier blocks. I thought this was weird enough already, so I just brushed it off, thinking this was a graphical bug. I then dropped down a few "Y" layers on top of another barrier block platform (that's when I noticed this probably was a problem). I couldn't get out with a scroll since I didn't have one on me. I started recording to get evidence. Sorry for my horrible english; I'm from a mostly french neighborhood. I'm also sorry for the horrible video quality; I don't have a screen recorder so this was the best I could do.

tl;dr: I think I am stuck until someone does something about it.

The evidence video won't fit in 10MB, so instead here's a link to that video:

https://www.youtube.com/watch?v=-tydYmeBJmY

Thanks for helping,
-tutuototo (Jonathan)

**Comment by LordHelmchen98** [comment]
Heyo @tutuototo,

what you experienced is an issue that mostly happens if you "put yourself in a 2-block high area". Unfortunately this causes issues as you spawn a bit higher than where you logged off and sometimes if there are blocks right above you you get teleported to the top of the block above you. In future I recommend to make sure there is at least one empty blocks above you.
The solution for your current problem is pretty simple. You can just do `/kill`. This will be like an actual death and you will get teleported to the closest city (which probably is Lusuco in your case). Do not use the command if you have 5 soul points or less because as it is like a actual death you will drop stuff and at 5 or less soul points you will start losing important stuff (check the chances by hovering over your soul points in your inventory).

Good luck.
